src/model/kaniRunner.ts

```diff
@@ -62,7 +62,7 @@ export async function runKaniCommand(
 	// Get cargo command and args for the command to be executed
 	const command = commmandSplit.commandPath;
 	const args = commmandSplit.args;
-	let kaniBinaryPath = '';
+	const kaniBinaryPath = '';
```

Comment:
I think you can remove this declaration altogether. The variable seems to be shadowed a few lines below.

Reply:
Thanks for pointing that out!